### PR TITLE
Moving while buckled or while in a locker that is locked / welded will no longer reset resist timer.

### PR DESCRIPTION
**Pull request by Spaghetti-bit**
## What Does This PR Do

Moving while buckled and cuffed (or while in a locked locker / welded locker) will no longer reset your unbuckle timer. This leaves the resist timer for breaking out of cuffs untouched.

## Why It's Good For The Game

Buckle-Cuffing in a moving object like a chair, roller-bed, or locker seemed a little broken and unintended especially when it was abusable. Just walking into the person in the object would reset them.
## Testing

Buckled a human to an office chair, moved them around. They were able to unbuckle themselves.
**Moved around a cuffed human, their timer was reset without fail.**
Moved around a locker that was welded, a person was inside, they were able to break out.
Moved around a secure locker, a person was inside, they were able to break out.
<hr>

### Declaration
- [x] I confirm that I either do not require [pre-approval](/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.

image

<hr>

## Changelog
:cl:
tweak: Moving while resisting in a movable object (or in a welded / locked locker) will no longer result in your buckle/resist timer being reset. 
/:cl:

### CODE REVIEW

**Comment by Daylight2** [comment]
The change to lockers seem reasonable, but I feel like moving around chairs is intentionally used to reset buckle timers quite often and not something that should be removed.

**Comment by Spaghetti-bit** [comment]
> The change to lockers seem reasonable, but I feel like moving around chairs is intentionally used to reset buckle timers quite often and not something that should be removed.

Just unbuckle and rebuckle.

**Comment by Fordoxia** [comment]
Holy mother of based.

This makes sense and I enjoy.

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 1]
```diff
@@ -358,7 +358,7 @@
 
 
 	spawn(0)
-		if(do_after(L,(breakout_time*60*10), target = src)) //minutes * 60seconds * 10deciseconds
+		if(do_after(L, breakout_time, target = src, allow_moving = TRUE, allow_moving_target = TRUE)) //minutes * 60seconds * 10deciseconds
```
Comment (Burzah) [inline code/game/objects/structures/crates_lockers/closets.dm:361]:
```suggestion
		if(do_after(L, breakout_time, target = src, allow_moving = TRUE, allow_moving_target = TRUE))
```

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 2]
```diff
@@ -340,7 +340,7 @@
 	return TRUE
 
 /obj/structure/closet/container_resist(mob/living/L)
-	var/breakout_time = 2 //2 minutes by default
+	var/breakout_time = 2 MINUTES //2 minutes by default
```
Comment (Burzah) [inline code/game/objects/structures/crates_lockers/closets.dm:343]:
```suggestion
	var/breakout_time = 2 MINUTES

File: code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm [hunk 3]
```diff
@@ -98,7 +98,7 @@
 		add_overlay("unlocked")
 
 /obj/structure/closet/secure_closet/container_resist(mob/living/L)
-	var/breakout_time = 2 //2 minutes by default
+	var/breakout_time = 2 MINUTES//2 minutes by default
```
Comment (Burzah) [inline code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm:101]:
```suggestion
	var/breakout_time = 2 MINUTES
```

File: code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm [hunk 4]
```diff
@@ -113,7 +113,7 @@
 
 
 	spawn(0)
-		if(do_after(usr,(breakout_time*60*10), target = src)) //minutes * 60seconds * 10deciseconds
+		if(do_after(usr, breakout_time, target = src, allow_moving = TRUE, allow_moving_target = TRUE)) //minutes * 60seconds * 10deciseconds
```
Comment (Burzah) [inline code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm:116]:
```suggestion
		if(do_after(usr, breakout_time, target = src, allow_moving = TRUE, allow_moving_target = TRUE)) 
```

**Review by Burzah — APPROVED** [review]

**Review by lewcc — APPROVED** [review]
LGTM

**Pull request merged**